packages/create-expo-app/src/dir.ts

```diff
@@ -0,0 +1,28 @@
+import { readdirSync } from 'fs';
+
+// Any of these files are allowed to exist in the projectRoot
+const tolerableFiles = [
+  // System
+  '.DS_Store',
+  'Thumbs.db',
+  // Git
+  '.git',
+  '.gitattributes',
+  '.gitignore',
+  // Project
+  '.npmignore',
+  '.travis.yml',
```

Comment:
tbh, with the [huge security breach from both Heroku and Travis CI](https://github.blog/2022-04-15-security-alert-stolen-oauth-user-tokens/), I'm not too keen to actively support Travis CI.